staging/src/k8s.io/client-go/transport/spdy/spdy.go

```diff
@@ -94,7 +86,6 @@ func Negotiate(upgrader Upgrader, client *http.Client, req *http.Request, protoc
 	if err != nil {
 		return nil, "", fmt.Errorf("error sending request: %v", err)
 	}
-	defer resp.Body.Close()
```

Comment:
Why was this removed?

Reply:
Same as above. Now, resp.Body is something that we want to leave open while we're streaming.

Reply:
Do we need to close the response body in error cases?